code/__defines/turfs.dm

```diff
@@ -19,4 +19,8 @@
 #define SMOOTH_ALL 1	//Smooth with all of type
 #define SMOOTH_WHITELIST 2	//Smooth with a whitelist of subtypes
 #define SMOOTH_BLACKLIST 3 //Smooth with all but a blacklist of subtypes
-#define SMOOTH_GREYLIST 4 // Use a whitelist and a blacklist at the same time. atom smoothing only
\ No newline at end of file
+#define SMOOTH_GREYLIST 4 // Use a whitelist and a blacklist at the same time. atom smoothing only
+
+
+#define DECK_HEIGHT 3	//3 metres in vertical height between decks
+//Note that the height of a turf's airspace is defined elsewhere as 2.5 metres, this adds extra to account for floor thickness
```

Comment:
Can't that be relative to original 2.5 meters?
Like `#define DECK_HEIGH (ATMOS_TILE_HEIGHT + 0.5)`.

Reply:
good idea, will do

Reply:
good idea, will do